GitCommands/StringExtensions.cs

```diff
@@ -232,5 +232,15 @@ public static string ShortenTo([CanBeNull] this string str, int maxLength)
 
             return str.Substring(0, maxLength - 3) + "...";
         }
+
+        /// <summary>
+        /// Uses the method IndexOf to check if the <paramref name="other"/> is contained inside the <paramref name="str"/>
```

Comment:
Minor, but documentation is more useful when it says what it does, not how it does it. I would ditch the part about IndexOf.

`String.Contains` can be a source of inspiration:

```c#
/// <summary>Returns a value indicating whether a specified substring occurs within this string.</summary>
/// <param name="value">The string to seek.</param>
/// <returns>true if the <paramref name="value" /> parameter occurs within this string, or if <paramref name="value" /> is the empty string (""); otherwise, false.</returns>
/// <exception cref="T:System.ArgumentNullException"><paramref name="value" /> is null.</exception>
public bool Contains(string value)
```

Reply:
Okay i changed the docs.